.github/actions/create-jira-ticket/action.yml

```
@@ -0,0 +1,118 @@
+name: Nightly/QG incident tracking
+description: Create Jira ticket on nightly build/veracode qg failure
+
+inputs:
+  jira_base_url:
+    required: true
+    description: jira base url
+  jira_user_email:
+    required: true
+    description: jira user email
+  jira_api_token:
+    required: true
+    description: jira api token
+  module_name:
+    required: true
+    description: module name
+  reason_for_ticket_creation:
+    required: true
+    description: reason for ticket creation
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+
+    - name: Get ticket elements from context
+      id: get_context
+      run: |
+        # Change context elements (summary, parent epic, etc.) that is checked depending on the reason the ticket is created
+        if [[ "${{ inputs.reason_for_ticket_creation }}" == "veracode" ]]; then
+          parent_epic_id=83818
+          parent_epic_key="AT-268"
+          ticket_summary="PR-${{ github.event.pull_request.number }} incident on ${{ inputs.module_name }}"
+          ticket_labels="[\"Veracode\", \"Pipeline\"]"
+          run_url="N/A"
+        elif [[ "${{ inputs.reason_for_ticket_creation }}" == "nightly" ]]; then
+          parent_epic_id=206242
+          parent_epic_key="MON-151547"
+          ticket_summary="$(date '+%Y-%m-%d') ${{ inputs.module_name }}-${{ github.ref_name }} nightly build failure"
+          ticket_labels="[\"Nightly\", \"Pipeline\"]"
+          run_url=$(gh run ls --json name,url --workflow web.yml --branch ${{ github.ref_name }} --limit 20 | \
+            jq --arg module_name "${{ inputs.module_name }}" --arg branch "${{ github.ref_name }}" '. | map(select(.name == ($module_name + " nightly " + $branch))) | .[0].url')
+          if [ $? -ne 0 ] || [ "$run_url" == "null" ]; then
+            echo "::error::Failed to get nightly run URL"
+            exit 1
+          fi
+        fi
+
+        echo "parent_epic_id=$parent_epic_id" >> $GITHUB_OUTPUT
+        echo "parent_epic_key=$parent_epic_key" >> $GITHUB_OUTPUT
+        echo "ticket_summary=$ticket_summary" >> $GITHUB_OUTPUT
+        echo "ticket_labels=$ticket_labels" >> $GITHUB_OUTPUT
+        echo "run_url=$run_url" >> $GITHUB_OUTPUT
+      shell: bash
+      env:
+        GH_TOKEN: ${{ github.token }}
+
+    - name: Check if the ticket already exists
+      id: check_ticket
+      run: |
+        # Checking if an incident ticket already exists
+        response=$(curl \
+          --request POST \
+          --url "${{ inputs.jira_base_url }}rest/api/3/search" \
```

A base url should not provide trailing slash
```suggestion
          --url "${{ inputs.jira_base_url }}/rest/api/3/search" \
```